catroid/src/main/java/org/catrobat/catroid/content/Look.java

```diff
@@ -431,10 +431,6 @@ public void setDirectionInUserInterfaceDimensionUnit(float degrees) {
 		}
 	}
 
-	public float getRealRotation() {
```

Comment:
it's a not used and misleading method which has the same return value as getDirectionInUserInterfaceDimensionUnit(). so it's duplicated code therefore I deleted it.